tutorial/02-materials.ipynb

```diff
@@ -629,10 +629,10 @@
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "Additionally, you may wish to convert the an existing set of atom fractions to a \n",
+      "Additionally, you may wish to convert an existing set of atom fractions to a \n",
       "new material stream.  This can be done with the `Material.from_atom_frac()` method, \n",
       "which will clear out the current contents of the material's composition and replace\n",
-      "it with the mass-weighted values.  Note that when you initialize a material from atom \n",
+      "them with the mass-weighted values.  Note that when you initialize a material from atom \n",
```

Comment:
`them` is an incorrect plural since we are talking about the "matrial's composition" (singular) rather than the "elements of the material's composition" as the object of what is being replaced.